### PR TITLE
Update test kit to include test annotations

### PR DESCRIPTION
**Pull request by khalp**
## 📃 Description

Update ComposeTesting to the new view system library that includes test annotations

## 🤔 Motivation and Context

To keep both library versions in the test kit in sync

## 🧪 How Has This Been Tested?

Added new **AnnotationTest.kt** file with tests

## ⚒️ Does the PR contain new tests or refactored old ones?

New tests

## 📦 Types of changes

- [x] New feature (non-breaking change which adds functionality)
- [ ] Enhancement to a current functionality
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Bug fix (non-breaking change which fixes an issue. Please add issue link here too).

## ✅ Checklist

- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] This PR contains new tests that cover the new code.
- [ ] This PR refactor previous tests to cover the new code.
- [ ] This PR is part of a set of PRs that build a bigger feature. If so, please, add here links to previously merged or open PRs.

### CODE REVIEW

File: ComposeTesting/sample/src/main/java/com/microsoft/device/dualscreen/testing/sample/MainActivity.kt [hunk 1]
```diff
@@ -36,6 +37,7 @@ class MainActivity : ComponentActivity() {
 @Composable
 fun ComposeTestingApp() {
     TwoPaneLayout(
+        paneMode = TwoPaneMode.HorizontalSingle,
```
Comment (joyl1216) [inline ComposeTesting/sample/src/main/java/com/microsoft/device/dualscreen/testing/sample/MainActivity.kt:40]: Why do we need this?

File: ComposeTesting/sample/src/androidTest/java/com/microsoft/device/dualscreen/testing/sample/AnnotationTest.kt [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    @Test
+    @SingleScreenTest(orientation = UiAutomation.ROTATION_FREEZE_0)
+    fun singlePortrait_showsOnePane() {
+        composeTestRule.onNodeWithText(composeTestRule.getString(R.string.pane1_text)).assertIsDisplayed()
+        composeTestRule.onNodeWithText(composeTestRule.getString(R.string.pane2_text)).assertDoesNotExist()
+    }
+
+    @Test
+    @DualScreenTest(orientation = UiAutomation.ROTATION_FREEZE_90)
+    fun dualLandscape_showsOnePane() {
```
Comment (joyl1216) [inline ComposeTesting/sample/src/androidTest/java/com/microsoft/device/dualscreen/testing/sample/AnnotationTest.kt:66]: Is there `showTwoPane` option for the `dualscreen?

**Comment by joyl1216** [comment]
Are we going to update README and document to provide all the available annotations?

File: ComposeTesting/sample/src/main/java/com/microsoft/device/dualscreen/testing/sample/MainActivity.kt (hunk 1 above)
Comment (khalp) [reply ComposeTesting/sample/src/main/java/com/microsoft/device/dualscreen/testing/sample/MainActivity.kt:40]: It's not necessary, but I changed it to make it easier to write some sample tests based on posture! For instance, dual portrait should be two panes but dual landscape would still be the single pane view

File: ComposeTesting/sample/src/androidTest/java/com/microsoft/device/dualscreen/testing/sample/AnnotationTest.kt (hunk 2 above)
Comment (khalp) [reply ComposeTesting/sample/src/androidTest/java/com/microsoft/device/dualscreen/testing/sample/AnnotationTest.kt:66]: Good point, I'll add a test for that case as well!

**Comment by khalp** [comment]
> Are we going to update README and document to provide all the available annotations?

Yes, I'll work on the README and Andrei is working on the [documentation PR](https://github.com/MicrosoftDocs/dual-screen-docs-pr/pull/362)

**Review by ancirja-m — APPROVED** [review]

**Review by joyl1216 — APPROVED** [review]

**Comment by joyl1216** [comment]
@khalp would you please also update the main README? 
1. update TwoPaneLayout release version
2. update ComposeTesting release version
3. update the social links for the TwoPaneLayout update

**Comment by joyl1216** [comment]
The latest update looks good, thanks!

**Pull request merged**